File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem21_UniqueNumberOfOccurrencesTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertTrue;

class Problem21_UniqueNumberOfOccurrencesTest {

    private final Problem21_UniqueNumberOfOccurrences test = new Problem21_UniqueNumberOfOccurrences();

    @Test
    void uniqueOccurrences() {
        int[] arr = new int[]{-3, 0, 1, -3, 1, 1, 1, -3, 10, 0};
        assertTrue(test.uniqueOccurrences(arr));
    }
}

File: JavaScript/leetcode75/src/test/Problem21_UniqueNumberOfOccurrences.test.js
const uniqueOccurrences = require("../main/Problem21_UniqueNumberOfOccurrences");

test("should return unique occurrences", () => {
  const arr = [1, 2, 2, 1, 1, 3];
  expect(uniqueOccurrences(arr)).toBe(true);
});
